src/navigation/mobileStepper/__test__/MobileStepper.test.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * To run this test:
+ * npx jest ./src/navigation/mobileStepper/__test__/MobileStepper.test.js
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import MobileStepper from '../MobileStepper';
+
+describe('<MobileStepper />', () => {
+    let wrapper;
+
+    const props = {
+        activeStep: 0,
+        backStepAction: console.log('Back'),
```

Comment:
Usually we recommend using `jest.fn()` for all required function props in component unit test fixtures.  This is a good noop by default, but also gives us good extensibility points if the test scenarios we want to cover include mocking these functions do particular things, and/or spying on them to assert (`expect()`) that they've been called under various circumstances.